pkg/k8shandler/forwarding_test.go

```diff
@@ -483,3 +487,94 @@ var _ = Describe("Normalizing Forwarding", func() {
 		})
 	})
 })
+
+func TestClusterLoggingRequest_generateCollectorConfig(t *testing.T) {
```

Comment:
Why not having this test also in Gingko/Gomega style?

Reply:
It's a simple unit test, why not to keep it as a simple test?

Reply:
In general we migrate tests to Gingko/Gomega where possible.

Reply:
In general, we should be moving towards Ginko as they are more expressive IMO as their intent.  Writing them should not be more difficult; it truly is just a different pattern over the existing test library.

Reply:
@jcantrill @periklis  Ok, but what's the advantage this Ginko gives to us over the standard table-driven testing?